src/winforms/toga_winforms/widgets/webview.py

```diff
@@ -1,13 +1,98 @@
+import sys
+
 from travertino.size import at_least
 
 from toga_winforms.libs import Uri, WinForms
 
 from .base import Widget
 
+ie_updated = False
+
+
+def _set_ie_mode():
+    """
+    By default hosted IE control emulates IE7 regardless which version of IE is installed. To fix this, a proper value
+    must be set for the executable.
+    See http://msdn.microsoft.com/en-us/library/ee330730%28v=vs.85%29.aspx#browser_emulation for details on this
+    behaviour.
+    """
+
+    global ie_updated
+
+    if ie_updated:
+        return
+
+    try:
+        import _winreg as winreg  # Python 2
+    except ImportError:
+        import winreg  # Python 3
+
+    def get_ie_mode():
+        """
+        Get the installed version of IE
+        :return:
+        """
+        ie_key = winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE,
+                                r"Software\Microsoft\Internet Explorer")
+        try:
+            version, type = winreg.QueryValueEx(ie_key, "svcVersion")
+        except Exception:
+            version, type = winreg.QueryValueEx(ie_key, "Version")
+
+        winreg.CloseKey(ie_key)
+
+        if version.startswith("11"):
+            value = 0x2AF9
+        elif version.startswith("10"):
+            value = 0x2711
+        elif version.startswith("9"):
+            value = 0x270F
+        elif version.startswith("8"):
+            value = 0x22B8
+        else:
+            value = 0x2AF9  # Set IE11 as default
```

Comment:
> By default hosted IE control emulates IE7 regardless which version of IE is installed

that means IE11 control works like IE7

now it emulates installed IE version.

Reply:
Yes, I understand that. That isn't what I asked.

I'm asking whether there is any reason why we should default to the behavior of the *installed* version of IE, rather than adopting the behavior of IE11 *always*.

Put another way - Is there *any* reasonable use case for adopting IE9 behavior?

Reply:
It will not work properly.
we cannot have IE11 features on  older versions.
I added this to
- to use best available standards
- to work properly with HTML5.